src/ch08/p1_count_syllables.py

```diff
@@ -16,7 +16,9 @@
 from src.ch02 import DICTIONARY_FILE_PATH
 from src.ch02.p1_cleanup_dictionary import cleanup_dict
 
-if not os.path.exists(os.path.expanduser('~/nltk_data/corpora/cmudict/cmudict')):
+if not os.path.exists(
+        os.path.expanduser('~/nltk_data/corpora/cmudict/cmudict')):
+    # pylint: disable=fixme
     # FIXME: This is nearly impossible to test.
     #  Patching os affects every use of os in the module.
     nltk.download('cmudict')
```

Comment:
Funnily enough, because the test environment doesn't have CMUdict installed, installing it counts as a test of this portion.

I'm leaving the `FIXME` because it's more of an unintended benefit rather than an actual test.